### PR TITLE
Add Icon: Paperless-ngx

### LINKED ISSUE #10287: Icon Request: Paperless-ngx
### Brand Name

Paperless-ngx

### Website

https://paperless-ngx.com/

### Popularity Metric

[Similarweb rank](https://www.similarweb.com/website/paperless-ngx.com/): 242,863
[Github stars](https://github.com/paperless-ngx/paperless-ngx): 14.8k

### Official Resources for Icon and Color

https://github.com/paperless-ngx/paperless-ngx/tree/dev/resources
https://github.com/paperless-ngx/paperless-ngx/tree/dev/resources/logo/web/svg
https://github.com/paperless-ngx/paperless-ngx/blob/dev/resources/logo/web/svg/square.svg

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by TheRealGramdalf**
**Issue:** closes #10287

**Similarweb rank:** 242,863

### Checklist

- [X] I updated the JSON data in `_data/simple-icons.json`
- [X] I optimized the icon with SVGO or SVGOMG
- [X] The SVG `viewbox` is `0 0 24 24`

### Description

_Each step was made in a separate commit for easier review_

- Hex value was taken directly from original SVG
- I was unable to find any license information regarding the original
- Paperless-ngx [is the official successor](https://github.com/paperless-ngx/paperless-ngx#paperless-ngx) to both Paperless-ng and the original, Paperless; they were added as backwards compatible names

Preview:
paperlessngx

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10625,6 +10625,17 @@
...
+        {
+            "title": "Paperless-ngx",
+            "hex": "17541F",
+            "source": "https://github.com/paperless-ngx/paperless-ngx/blob/e16645b146da24f07004eb772a455450354a37a7/resources/logo/web/svg/square.svg",
+            "aliases": {
+                "old": [
+                    "paperless",
+                    "paperless-ng"
+                ]
+            }
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:10637]:
```suggestion
            "source": "https://github.com/paperless-ngx/paperless-ngx/blob/e16645b146da24f07004eb772a455450354a37a7/resources/logo/web/svg/square.svg"
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks for the contribution, @TheRealGramdalf 👍🏻 However, I'm seeing a couple of small differences that will need to be cleaned up, most notably in [the cut-out of the left leaf](https://github.com/simple-icons/simple-icons/assets/15157491/0f9aa16a-44f0-487d-acc9-243f10ccc300). This _could_ just be a precision issue.

Also, as both aliases are contained within the `title` there's no need to include them here.

**Comment by TheRealGramdalf** [comment]
> Thanks for the contribution, @TheRealGramdalf 👍🏻 However, I'm seeing a couple of small differences that will need to be cleaned up, most notably in [the cut-out of the left leaf](https://github.com/simple-icons/simple-icons/assets/15157491/0f9aa16a-44f0-487d-acc9-243f10ccc300). This _could_ just be a precision issue.

The cut out is actually part of the [official logo](https://github.com/paperless-ngx/paperless-ngx/blob/dc1918ad10436bc05f519f588ae60c4b5204763f/resources/logo/web/svg/square.svg) - it looks like it was originally intended for there to be a second notch in the larger leaf (there were two nodes that looked to be forming a notch that weren't colored correctly in the original). 

> Also, as both aliases are contained within the `title` there's no need to include them here.

This would still present issues with client libraries, no? I'm not familiar with the inner workings, but I don't think you can use part of a name to resolve it - i.e. if I specify the icon `si-paperless` in dashy (where `si-` is the simple icons slug), it wouldn't resolve to the PNGX logo unless `paperless` is listed as an alias

**Comment by PeterShaggyNoble** [comment]
I wasn't suggesting removing the cut-out above, @TheRealGramdalf, simply using it to illustrate the small differences between your version (which is in red [the image I linked](https://github.com/simple-icons/simple-icons/assets/15157491/0f9aa16a-44f0-487d-acc9-243f10ccc300) and the source.

On the aliases: they're not used to retrieve icons; the `old` ones in particular are intended to be used for discoverability purposes. For example, to allow people to find icons on our website by searching for them by their old name.

**Comment by TheRealGramdalf** [comment]
Got it. I believe the cutout mismatch is due to the stroke width - I removed that so there was a single path/fill. I'm not quite sure how to fix that precisely, I'm unfortunately not an SVG wizard (I primarily specialize in sysadmin).

**Comment by PeterShaggyNoble** [comment]
I'm still seeing those same differences, @TheRealGramdalf. Your mention of a `stroke`, though, makes me wonder if you're not working from a different source file than the one you have in the JSON here, which doesn't have a `stroke`. So I gave it a try myself and came up with the following, if you'd like to run with it:

```
M6.338 23.028c-.117-.56-.353-1.678-.382-1.678-4.977-2.975-4.388-8.128-2.739-11.073.353 3.71 6.92 6.273 3.092 10.808-.03.059.177.765.353 1.413.766-1.296 1.915-2.856 1.856-3.004C3.806 8.01 18.53 7.126 21.592 0c1.385 6.89-.706 17.55-12.544 20.26-.06.03-2.15 3.71-2.238 3.74 0-.059-.884-.03-.766-.324.059-.177.177-.412.294-.648zm-.147-2.768c1.502-1.737-.265-4.712-1.325-5.683 1.796 3.092 1.679 4.888 1.325 5.683z
```

**Comment by TheRealGramdalf** [comment]
> I'm still seeing those same differences, @TheRealGramdalf.

Apologies if it wasn't clear - I'm not sure how to fix that myself (i.e. what values to tweak), since I don't do SVG work very often.

>Your mention of a `stroke`, though, makes me wonder if you're not working from a different source file than the one you have in the JSON here, which doesn't have a `stroke`. 

I'm referring to this: https://github.com/simple-icons/simple-icons/blob/36bdd3aabd2b699dc1b770f0a6c21a21ed597994/icons/paperlessngx.svg?short_path=0fec83c
It looks like exporting from inkscape to a plain SVG added the stroke value, which I guess is represented differently with the original [inkscape version](https://github.com/paperless-ngx/paperless-ngx/blob/dc1918ad10436bc05f519f588ae60c4b5204763f/resources/logo/web/svg/square.svg?short_path=3144ae9)

>So I gave it a try myself and came up with the following, if you'd like to run with it:
> 
> ```
> M6.338 23.028c-.117-.56-.353-1.678-.382-1.678-4.977-2.975-4.388-8.128-2.739-11.073.353 3.71 6.92 6.273 3.092 10.808-.03.059.177.765.353 1.413.766-1.296 1.915-2.856 1.856-3.004C3.806 8.01 18.53 7.126 21.592 0c1.385 6.89-.706 17.55-12.544 20.26-.06.03-2.15 3.71-2.238 3.74 0-.059-.884-.03-.766-.324.059-.177.177-.412.294-.648zm-.147-2.768c1.502-1.737-.265-4.712-1.325-5.683 1.796 3.092 1.679 4.888 1.325 5.683z
> ```

I gave it a look and that looks fine. I added the changes and rebased, so everything should be good to go.

**Review by service-paradis — APPROVED** [review]
LGTM 🎉 
Thank you @TheRealGramdalf and @PeterShaggyNoble!

**Pull request merged**